internal/pkg/cli/flag.go

```diff
@@ -64,6 +64,16 @@ const (
 	securityGroupsFlag = "security-groups"
 	envVarsFlag        = "env-vars"
 	commandFlag        = "command"
+
+	vpcIDFlag          = "import-vpc-id"
+	publicSubnetsFlag  = "import-public-subnets"
+	privateSubnetsFlag = "import-private-subnets"
+
+	vpcCIDRFlag            = "override-vpc-cidr"
+	publicSubnetCIDRsFlag  = "override-public-cidrs"
+	privateSubnetCIDRsFlag = "override-private-cidrs"
+
+	defaultEnvironmentFlag = "use-default"
```

Comment:
I wonder if this should be `--no-custom-resources`? I'm worried that things might get confusing with credential flags also coming to `env init` and `--use-default` might make it seem like use the default credential chain and default env.

Reply:
yeah that makes sense! i'll update it